.github/workflows/fbgemmci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         make
 
     - name: Test static FBGEMM lib
-      if: contains(runner.os, 'Linux')   # not run on macos-latest now due to supporting AVX2
+      if: contains(runner.os, 'linux')   # not run on macos-latest now due to supporting AVX2
```

Comment:
contains(runner.os, 'Linux') doesn't work?

Reply:
It works. The purpose here is to run Test only on Linux platforms now, not macos. Macos-latest does not support AVX2.

Reply:
Yes I understand the purpose of this line. I was just curious about why you changed Linux to linux

Reply:
It's not case sensitive. Just to be consistent with others.